src/components/EndreInntekt/EndreInntekt.tsx

```diff
@@ -78,7 +78,8 @@ export const EndreInntekt: React.FC<Props> = ({
     const tekst = intl.formatMessage({
       id,
     })
-    logger('valideringsfeil', {
+    logger('skjema validering feilet', {
+      skjemanavn: 'endre-inntekt',
```

Comment:
Samme her. SkjemaId bør helst ikke være hardkodet. Hvis ikke det finnes noe variabel på det fra før av kan du følge samme mønster som for de andre skjemaene og opprette det som trengs.